cuda/extensions.bzl

```diff
@@ -94,6 +94,9 @@ def _module_tag_to_dict(t):
 
 def _redist_json_impl(module_ctx, attr):
     url, json_object = redist_json_helper.get(module_ctx, attr)
+    # Sets the default to be all the components contained if not specified.
+    if attr.components == None:
+        attr.components = json_object.keys()
```

Comment:
keys are not available.

We have builtin mapping for it, see registry.bzl

Reply:
I've had another go which attempts to match components you specify in the `registry.bzl`